modules/tinymce/src/themes/silver/main/ts/ui/core/complex/StyleFormat.ts

```diff
@@ -81,7 +81,7 @@ const mapFormats = (userFormats: AllowedFormat[]): CustomFormatMapping => Arr.fo
     };
   } else if (isInlineFormat(fmt) || isBlockFormat(fmt) || isSelectorFormat(fmt)) {
     // Convert the format to a reference and add the original to the custom formats to be registered
-    const formatName = `custom-${fmt.title.toLowerCase()}`;
+    const formatName = Type.isString(fmt.name) ? fmt.name : `custom-${fmt.title.toLowerCase()}`;
```

Comment:
How concerned are we that people will get confused and this will override normal formats (as that would be unexpected for me personally)? I'm thinking it might be better to always keep the `custom-` prefix here.

Reply:
I prefixed the formats with `custom-` but I guess in theory it would still override any format called `custom-` but I guess that it less likely.

Reply:
Would it be worth adding a note in the docs about this?